app/code/Magento/Review/Model/Rss.php

```diff
@@ -6,10 +6,11 @@
 namespace Magento\Review\Model;
 
 /**
- * Class Rss
- * @package Magento\Catalog\Model\Rss\Product
+ * Rss Model
+ *
+ * Class \Magento\Catalog\Model\Rss\Product\Rss
  */
-class Rss extends \Magento\Framework\Model\AbstractModel
+class Rss
```

Comment:
Unfortunately, we can't remove inheritance according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/). Someone can use Rss model in 3d party code and use methods from the `\Magento\Framework\Model\AbstractModel`
It's better to call the parent constructor here, looks like that call was missed during the refactoring.

Reply:
@sidolov: I have changed it. Now, I think it following the Backward Compatible Development 
Guide. 
Thank you very much.